stackabletech/test-dev-cluster
# Feature/multiple agent containers

### Description

Currently only one agent container is set up when testing an operator. This is a serious limitation for some operators.
Extend the initialization step to start multiple agents. Ideally provide a command line parameter to init.sh for this purpose.

### How to test

Start a cluster with two kubelets:

`./init.sh debian zookeeper-operator --scale agent=2`

Check that two nodes are registered with k3s:

`docker exec -t k3s kubectl get nodes`
